cep-6.md

```diff
@@ -0,0 +1,215 @@
+<table>
+    <tr><td> Title </td><td>Add Channel Notifications to conda</td>
+    <tr><td> Status </td><td>Draft</td></tr>
+    <tr><td> Author(s) </td><td>Travis Hathaway &lt;thathaway@anaconda.com&gt;</td></tr>
+    <tr><td> Created </td><td> Apr 21, 2022</td></tr>
+    <tr><td> Updated </td><td> Aug 21, 2022</td></tr>
+    <tr><td> Discussion </td><td>NA</td></tr>
+    <tr><td> Implementation </td><td>NA</td></tr>
+</table>
+
+
+## Abstract
+
+In order to facilitate better communication between users of `conda` and channel owners, we want to 
+implement a notification feature.
+Specifically, this feature will give channel owners the ability to send users a small message (about the size 
+of a single Twitter post). How often this message displays will be set by channel owners
+and will be displayed for each channel a user is actively using (i.e. everything under "channels" in a
+user's `.condarc`).
+
+The messages may contain but are not limited to the following use cases:
+
+- News about the latest package additions to the channel
+- How to sponsor, volunteer or help out a particular channel
+- Warnings about a breach of TOS while using a particular channel
+- Status updates about the stability of the channel
+
+
+## Specification
+
+The specification is described below via a question-answer format.
+
+### When will this message be shown?
+
+The notification message will appear while running the following commands:
+
+- `create`
+- `env create`
+- `install`
+- `update`
+- `env update`
+- `search`
+
+The reasoning behind this decision is that the above commands all retrieve `repodata.json` from the configured
+channels. Simply adding another file request (which is very small and immediately cached) would not add much more
+overhead to the execution of these commands. Additionally, these are commands where our users already expect network
+traffic to occur, so the requirement for having an active internet connection is assumed.
+
+Here's an example of how a notification message may appear while running the `conda create` command:
+
+```
+$ conda create -n geopandas geopandas
+Collecting package metadata (repodata.json): done
+Solving environment: done
+
+## Package Plan ##
+
+  environment location: /Users/username/opt/conda_x86_64/envs/geopandas
+
+  added / updated specs:
+    - geopandas
+
+... (output truncated)
+
+Notice [info]:
+> Here is a message to the user
+> Here is a link they could click: https://example.com/link-name
+> To see the message again, run `conda alert`
+
+```
+
+### How else can our users access this message?
+
+Additionally, because our users may wish to see this message on demand, we will add a new sub-command called `alerts`. 
```

Comment:
```suggestion
Additionally, because our users may wish to see this message on demand, we will add a new sub-command called `notices`. 
```

I'll comment on this later more, but I think given the wide range of use cases for channel notifications we shouldn't call this subcommand `alerts` since it predefines a level of alarm that may not actually be matching the messages intentions.

I suggest to instead use (and rename the proposal to): `notices`